### PR TITLE
My Lesson 1 & 2 Task 4 done 

### PR DESCRIPTION
**Pull request by iamipheco**
@promise-J
@ChidiPatrick
@OgJaykay
@Pieta9221
@MiniHack
@Melumm

I'm done with my Lesson 1 & 2 Task 4, Please Kindly Check it out.

### CODE REVIEW

**Review by OgJaykay — COMMENTED** [review]
CHECK YOUR READ ME FILE AND UPDATE
CHANGE YOUR HTML FOLDER NAME TO HTML NOT DOCS

**Comment by iamipheco** [comment]
> 

About the Readme. md file, Am on it right now to get it updated but for the Docs, we were instructed by our tutor about that

**Comment by OgJaykay** [comment]
@promise-J please are you aware of the Docs folder naming?
If your are done editing you can merge, good work👍

**Comment by iamipheco** [comment]
> @promise-J please are you aware of the Docs folder naming? If your are done editing you can merge, good work👍

Alright thanks

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Ipheco Chris Blog</title>
+    </head>
+    <body>
+        <section>
+            <h1>The Power of Meditation for a Healthier You</h1>
+            <img src="https://vegan.org/wp-content/uploads/2019/10/Meditation-Stock-600x492.jpg" alt="meditation image">
```
Comment (ChidiPatrick) [inline docs/index.html:12]: Close the image tag

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                numerous health benefits. From reducing stress and anxiety to
+                improving sleep and boosting immunity, meditation is a powerful
+                tool that can help you live a healthier, happier life.
+            </p>
+
+            <p>
+                Do you want to read on more article on the above topics on
+                meditation, kindly click on any of these link below
+            </p>
+            <a name="top"></a>
```
Comment (ChidiPatrick) [inline docs/index.html:26]: What does this anchor element do? It doesn't have href attribute  and content in between. If it's for easy navigation up and down the page, it should have an id attribute that is linked to another element  with target attribute value of the form #yourChosenId

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                    by helping reduce cravings and increasing feelings of
+                    self-control and self-awareness.
+                </p>
+            </ol>
+            <p>
+                For more detailed information on Meditation, please see links
+                <a href="#">Back to top</a>
+            </p>
+        </section>
+        <a name="bottom"></a>
```
Comment (ChidiPatrick) [inline docs/index.html:152]: Same correction here

**Review by ChidiPatrick — COMMENTED** [review]
Implement the changes and lemme know if it's working the way it should

**Pull request merged**